src/stratigraphy/util/language_detection.py

```diff
@@ -36,8 +36,8 @@ def detect_language_of_document(doc: fitz.Document) -> str:
     try:
         language = detect(text)
     except LangDetectException:
-        language = "de"
+        language = "de"  # TODO: default language should be read from config
```

Comment:
This is bothering me for the moment. Right now you need to adjust the code to extend the extraction to other languages. This should be doable from the config files. I believe there are other places where language is hard-coded in form of keywords. (e.g. coordinate extraction)

 I will open an issue for it.